### DISCUSSION TITLE
Make egg features a tag

### DISCUSSION THREAD

**Discussion by parkervcp** [pelican-dev/panel#143] [Ideas]
From a side conversation, egg features can be a tag instead of manually being added to the egg like they are now.

**Comment by moto6333** [comment]
@parkervcp is this what was talked about in the most recent release?

`~ Nests, To also be replaced with Egg Tags`

Is there already something in progress for this? Otherwise I would be interested in adding this.

**Reply by parkervcp** [reply]
Already being worked on

**Comment by lancepioch** [comment]
Thanks, please follow: https://github.com/pelican-dev/panel/issues/222
